### ISSUE TITLE
llvm-bolt-heatmap interprets executable as other argument

### ISSUE THREAD

**Issue by peterwaller-arm** [llvm/llvm-project#53633]
`llvm-bolt heatmap` appears to have broken at some point. I've never succesfully used it yet, I thought I'd give it a quick whirl but I get:

```
bin/llvm-bolt-heatmap -p perf.data foobar-executable
llvm-bolt-heatmap: Unknown command line argument 'foobar-executable'.  Try: 'bin/llvm-bolt-heatmap --help'
```

```
bin/llvm-bolt-heatmap -p perf.data
bin/llvm-bolt-heatmap: '': No such file or directory.
```

The help output looks like this, and it doesn't list `<executable>` as an input option, as the source code suggests should be the case:

```
$ bin/llvm-bolt-heatmap --help
SUBCOMMAND 'heatmap': generate heatmap

USAGE: llvm-bolt-heatmap heatmap [options]

OPTIONS:

BOLT generic options:

  -v=<uint>             - set verbosity level for diagnostic output

Data aggregation options:

  --perfdata=<string>   - <data file>

General options:

  --block-size=<uint>   - size of a heat map block in bytes (default 64)
  --line-size=<uint>    - number of entries per line (default 256)
  --max-address=<ulong> - maximum address considered valid for heatmap (default 4GB)
  --min-address=<ulong> - minimum address considered valid for heatmap (default 0)
  -o=<string>           - heatmap output file (default stdout)

Generic Options:

  --help                - Display available options (--help-hidden for more)
  --help-list           - Display list of available options (--help-list-hidden for more)
```

/cc @maksfb

**Comment by maksfb** [comment]
Thank you for letting know. Indeed, the heatmap was broken recently. The fix is on the way (D118946).

**Comment by yota9** [comment]
Hello @peterwaller-arm ! The D118946 has been landed. I hope you won't mind if I will close the issue, please re-open in case of any problems, thank you :)
